src/lib.rs

```diff
@@ -457,6 +488,7 @@ mod tests {
                     debug_mode: false,
                     size: Size::Small,
                     provided: Some(String::from("test")),
+                    newtype: CustomNewType(String::from("42"))
```

Comment:
fancy!

Reply:
I just made it fancier by *not* using a `String` in the newtype, so we now test that the inner time gets properly serialized as well.